packages/blade/src/utils/isReactNative/castUtils.ts

```diff
@@ -0,0 +1,11 @@
+import type { Platform } from './';
+
+const castWeb = <T>(value: T): Platform.CastWeb<T> => {
```

Comment:
Lets call this `castWebType` & `castNativeType` to be more explicit on what "casting" is happening here. Folks unaware of the context might get confused why certain things are wrapped around castWeb. 

Also can you add proper description her in comments for both `castWebType` & `castNativeType` so that if anyone visits these files will know what these functions do in detail? Its a fairly complex thing to understand with fresh eyes

Reply:
Done.